tests/nnapi/specs/Ex/matrix_band_part_ex_4D_float_nnfw.mod.py

```diff
@@ -0,0 +1,38 @@
+# model
```

Comment:
`_nnfw` suffix is not need for `_EX` operation.

Reply:
I removed the suffix.